dev-util/cppunit/cppunit-1.14.0.recipe

```diff
@@ -11,27 +10,33 @@ COPYRIGHT="2000 Jerome Lacoste
 	2001-2002, 2007-2008 Steve Robbins
 	2012-2013 Harvey Brydon"
 LICENSE="GNU LGPL v2.1"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="http://dev-www.libreoffice.org/src/cppunit-$portVersion.tar.gz"
```

Comment:
```diff
-SOURCE_URI="http://dev-www.libreoffice.org/src/cppunit-$portVersion.tar.gz"
+SOURCE_URI="http://dev-www.libreoffice.org/src/cppunit-$portVersion.tar.gz"
```

Reply:
@alaviss, I guess you meant:
```diff
-SOURCE_URI="http://dev-www.libreoffice.org/src/cppunit-$portVersion.tar.gz"
+SOURCE_URI="https://dev-www.libreoffice.org/src/cppunit-$portVersion.tar.gz"
```
(You forgot to add the "s" after the copy paste.)